### PR TITLE
Cc 33 alpha beta gamma radiation

### PR DESCRIPTION
**Pull request by tyler-barry**

### CODE REVIEW

File: Physics/CC-33-Radiation-Animations/Beta_minus/Beta_minus.pde [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+  float[][] coords = {{(x_val),(y_val-(2*radius))}, {(x_val+x_dist),(y_val-y_dist)}, {(x_val+x_dist),(y_val+y_dist)}, {(x_val),(y_val+(2*radius))}, {(x_val-x_dist),(y_val+y_dist)}, {(x_val-x_dist),(y_val-y_dist)}};
+  
+  return coords;
+}
+
+Mover[] movers = {};
+
+
+//function to check if a coordinate is already used
+boolean isItemInArray(float[][] array, float[] item) {
```
Comment (arcotbha) [inline Physics/CC-33-Radiation-Animations/Beta_minus/Beta_minus.pde]: Functions like these which are being used at multiple places can be moved to a separate helper class and referenced from there.

File: Physics/CC-33-Radiation-Animations/Beta_plus/Beta_plus.pde [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+    else {
+      fill(12,48,224);
+    }
+    ellipse(location.x,location.y,rad,rad);
+  }
+}
+
+
+// new function
+float[][] position(float x_val, float y_val, float radius) {
```
Comment (arcotbha) [inline Physics/CC-33-Radiation-Animations/Beta_plus/Beta_plus.pde]: This function can also be moved to a helper class.

**Comment by cmann** [comment]
I've got a working notebook on a separate branch (CC-33-Alpha-Beta-Gamma-Radiation-updated) with most of the repeated code refactored to a shared library. Similar to Preethi I found that the processing code wasn't running properly. After fixing paths there were problems with iFrames and CORS and loading the Processing code. I removed the wrapping iFrames and just got empty output with no relevant errors showing up the browser's developer console. Finally I used the same technique as the StatiscalReasoning notebook and embedded the Processing code in the notebook in Python strings which is working now.

If this is an acceptable solution I'll merge my branch into this one, otherwise I could use some help troubleshooting why loading the pde files as external resources doesn't work from someone with more experience with JupyterHub.

**Comment by cmann** [comment]
Pushed previously mentioned changes and rebased the branch on top of staging to bring it up to date. Also noticed the bottom banner was broken so fixed that too.

**Comment by cmann** [comment]
Fixed the broken animations that Preethi pointed out.

**Pull request merged**